makefile

```diff
@@ -27,5 +23,5 @@ deploy: $(APPNAME)
 	echo $$MINOR ;\
 	echo $$BUILD_NUM ;\
 	echo $$BUILD_HASH ;\
+	sed -i "s/ComponentLibrary(\s*)id=\"TruexAdRendererLib\"(\s*)uri=\".*\"/ComponentLibrary(\s*)id=\"TruexAdRendererLib\"(\s*)uri=\"http://@{S3_BUCKET}roku/v${MAJOR}_${MINOR}/${RC_DEVELOP}/${APPNAME}-${RC_DEVELOP}-v${MAJOR}.${MINOR}.${BUILD_NUM}-${BUILD_HASH}.zip"/g" components/MainScene.xml ;\
```

Comment:
This should be using the signed PKG file, not the unsigned ZIP file.

Reply:
Oops! Updated in c9fa5f7